src/components/MeetingTypes/MeetingTypeCard.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import { Card, Button, Tag, Typography, notification, Icon } from "antd";
+import "./style.scss";
+
+import EditMeetingTypeForm from "./EditMeetingTypeForm";
+import { connect } from "react-redux";
+
+class MeetingTypeCard extends Component {
+  state = {
+    editing: false,
+    loading: false,
+  };
+
+  handleFormOpen = () => {
+    this.setState({ editing: !this.state.editing });
+  };
+
+  getNameFromIconFlags = (iconData) => {
+    if (!iconData) return;
+
+    const iconText = this.props.iconFlags.find(
+      (icon) => icon.data == iconData.toString()
+    );
+
+    return iconText ? iconText.text : "";
+  };
+
+  openNotification = (title, description) => {
+    notification.open({
+      message: title,
+      description,
+      icon: <Icon type="smile" style={{ color: "#108ee9" }} />,
+      duration: 3.0,
+    });
+  };
+
+  render() {
+    const { Text } = Typography;
+
+    const { name, description, icon, color, display } = this.props.meetingType;
```

Comment:
```suggestion
        const { name, description, icon, color, display, text, email } = this.props.meetingType;
```

Reply:
just missing the tags for the other 2 checkboxes

Reply:
done